python/lsst/ip/isr/isrMockLSST.py

```diff
@@ -68,6 +70,26 @@ class IsrMockLSSTConfig(IsrMockConfig):
         default=30000.0,
         doc="Bright defect level (electron).",
     )
+    doAddSiliconPhysics = pexConfig.Field(
+        dtype=bool,
+        default=False,
+        doc="Add brighter fatter and/or diffusion effects to image.",
+    )
+    bfStrength = pexConfig.Field(
+        dtype=float,
+        default=1.0,
+        doc="The brighter fatter effect scaling parameter (cannot be zero)."
+    )
+    diffusionStrength = pexConfig.Field(
+        dtype=float,
+        default=1.0,
+        doc="The diffusion effect scaling parameter."
+    )
+    nRecalc = pexConfig.Field(
+        dtype=int,
+        default=10000,
+        doc="Number of e- to accumulate before recalculating pixel shapes.",
```

Comment:
We have been standardizing on using the proper unit name `electron` in these places.  (And `adu` where appropriate).

Reply:
Is this to do with the silicon physics?  It maybe should be renamed to be clear.

Reply:
Took away the diffusion part since it is unused anyways; renamed to `doAddBrighterFatter`.